src/components/FileList/filelist.css

```diff
@@ -1,11 +1,39 @@
 /* FileList and FileListElement CSS */
 .FileList {
-  @inherit: .list-reset, .min-w-full, .my-20, .border-t, .border-grey;
+  @inherit: .list-reset, .min-w-full, .my-20, .p-0, .ml-0, .border-t,
+    .border-grey;
 }
 
 .FileList--Element {
-  @inherit: .list-reset, .w-full, .border-b, .border-grey, .py-8, .px-32, .flex,
-    .flex-row, .flex-wrap-reverse;
+  @inherit: .list-reset, .w-full, .py-8, .border-b, .border-grey, .text-black;
+  transition: all 0.25s ease-in;
```

Comment:
The animation is pretty frustrating and having the description hidden makes it difficult to scan the files for the one you one and compare across them.
Are there other approaches for showing the description that allows that?

Reply:
sped the animation up. If/when we get pagination in place we could potentially have the truncated desc without the hover state. As of now I think that showing the file desc is adding too much to clutter to a simple list item